helpers/focus.js

```diff
@@ -37,13 +37,13 @@ export function getComposedActiveElement() {
 	return node;
 }
 
-export function getFirstFocusableDescendant(node, includeHidden, predicate) {
+export function getFirstFocusableDescendant(node, includeHidden, predicate, includeTabbablesOnly) {
 	if (predicate === undefined) predicate = () => true;
 
 	const composedChildren = getComposedChildren(node);
 
 	for (let i = 0; i < composedChildren.length; i++) {
-		if (isFocusable(composedChildren[i], includeHidden) && predicate(composedChildren[i])) return composedChildren[i];
+		if (isFocusable(composedChildren[i], includeHidden, includeTabbablesOnly) && predicate(composedChildren[i])) return composedChildren[i];
 
 		const focusable = getFirstFocusableDescendant(composedChildren[i], includeHidden, predicate);
```

Comment:
This method is recursive and calls itself, so I think we'd want `includeTabbablesOnly` passed in here as well, yes?